src/components/elements/carousel/carousel.js

```diff
@@ -65,10 +77,12 @@ export const Carousel = ({
     has_autoplay,
     autoplay_interval,
     vertical_container,
+    navigation_style,
```

Comment:
Can you help to sort these in alphabetical order. So it's easier to find a prop later on 🙏🏻